scripts/soltest.sh

```diff
@@ -23,7 +43,11 @@ do
 			shift
 			BOOST_OPTIONS="${BOOST_OPTIONS} $1"
 			;;
-		-t)
+		--help)
+		        usage
+			exit 100
```

Comment:
Actually why does this use exit code 100?

Reply:
This has been my convention (going way back to the days of mainframes). Some people like to distinguish getting help from other kinds of activities.

I don't have a strong opinion about this, so if this is offensive, we can remove it.

Reply:
Not offensive, just inconsistent with the rest of the codebase.

Reply:
Ok. Change to 0 now.